### PR TITLE
Add gds-dmt and gds-monitor-apps

### PR DESCRIPTION
**Pull request by duncanmmacleod**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@emaros, please confirm you are happy to be named as co-maintainer of this recipe.

**Comment by emaros** [comment]
> @emaros, please confirm you are happy to be named as co-maintainer of this recipe.

I agree to being named as co-maintainer

**Comment by duncanmmacleod** [comment]
Am not entirely sure who is forcing `libcxx >=15.0.7` into the host env spec, but it is conflicting with root_base. @chrisburr, is it easier to work around this here and pin to a lower `libcxx` version (how to do that escapes me), or to just update `root` to build and link against clang 15?

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this recipe is ready for review, thanks.

File: recipes/gds-monitor-apps/build.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+mkdir -p _build
+cd _build
+
+# set C++ standard to c++14
+#   - igwn-cmake sets c++11, which is too old for root
+if [ "$(uname)" = "Darwin" ]; then
```
Comment (xhochy) [inline recipes/gds-monitor-apps/build.sh:10]:
```suggestion
if [[ "$target_platform" == osx-* ]]; then
```

**Review by xhochy — CHANGES_REQUESTED** [review]
One nit otherwise lgrm

**Comment by duncanmmacleod** [comment]
@xhochy, thanks for the review, I missed the notification when it came in, but I have added your suggested change now. If you have time to review afresh, thanks!

**Review by xhochy — APPROVED** [review]

**Pull request merged**